Objects/object.c

```diff
@@ -1712,6 +1712,29 @@ notimplemented_dealloc(PyObject* ignore)
     Py_FatalError("deallocating NotImplemented");
 }
 
+static int
+notimplemented_bool(PyObject *v)
+{
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+                     "NotImplemented should not be used in a boolean context",
+                     1) < 0)
+        return -1;
```

Comment:
Add braces. It is hard to read the code with multiline condition in `if`.
```c
   if (PyErr_WarnEx(PyExc_DeprecationWarning,
                     "NotImplemented should not be used in a boolean context",
                     1) < 0)
    {
        return -1;
    }
    return 1;
```

Reply:
Fine by me; I try to copy existing style, and I was following the majority usage in Objects/*.c (PyErr_WarnEx is called in 10 places, and 8 of the 10 don't use braces), but I agree the braces are probably for the best.

Reply:
Done in 516f31f.